app/main.py

```diff
@@ -1,6 +1,24 @@
+import requests
+from api_key import API_KEY
+
+URL = "http://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    weather = requests.get(URL + f"?key={API_KEY}" + f"&q={FILTERING}").json()
```

Comment:
Ensure that the API key is retrieved from an environment variable instead of being hardcoded or imported from a file with hardcoded values.